### PR TITLE
{lib}[foss/2019a] libspatialite v4.3.0a, FreeXL v1.0.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3135.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/82bb2ed00abaed197877a7bc54a0a860 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2085.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b2501c44dff2167d00a19a8bf739468e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2485.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9f913c1a855f9160786f5958b650643d for a full test report.

File: easybuild/easyconfigs/l/libspatialite/libspatialite-4.3.0a_depr-PROJ-API.patch [hunk 1]
```diff
@@ -0,0 +1,23 @@
+--- libspatialite-4.3.0a/configure.orig	2015-09-07 15:56:45.000000000 +0200
```
Comment (Micket) [inline easybuild/easyconfigs/l/libspatialite/libspatialite-4.3.0a_depr-PROJ-API.patch]: Description and author

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/1c38ada1f7f37cbc854daed14613bd70 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3116.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/b26807ce56b97b9293495513ea4cb932 for a full test report.

**Comment by boegel** [comment]
@Micket Good to go now?

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**